### DISCUSSION TITLE
Cannot install Compose extension on desktop podman

### DISCUSSION THREAD

**Discussion by blazerodri** [containers/podman#23794] [Q&A]
When on Podman, to add the extension "Compose", one clicks on it, then on set-up, but it fails and there is no helpful error message. I simply shows "failed, try again".
If I try via PowerShell is says "PATH NOT FOUND" for docker-compose, but I've checked with other people I know that have it working and I have the exact same files and specifications.
Any idea what might be happening ?

**Comment by Luap99** [comment]
We do not maintain desktop here, if you have problems with podman-desktop you need to ask there: https://github.com/containers/podman-desktop
